tests/integration_tests/test_sim.py

```diff
@@ -103,21 +103,18 @@ def test_gps_read(self, qtbot, sim_app, device_type):
             assert sim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.GPS_ALTITUDE) == vals[2]
 
     def test_pin_mode(self, qtbot, sim_app, device_type):
-        test_vals = [
-            (1, 1),
-            (12, 0),
-            (5, 0),
-            (1, 0),
-            (2, 1),
-        ]
-
         hw = get_hw_sim(sim_app, device_type)
-        for vals in test_vals:
-            hw.set_pin_mode(vals[0], vals[1])
-            flush_packets(sim_app, device_type)
-
-            assert hw.get_pin_mode(vals[0]) == vals[1]
 
+        assert hw.get_pin_mode(20) == PinModes.INPUT
+        assert hw.get_pin_mode(21) == PinModes.INPUT
+        assert hw.get_pin_mode(13) == PinModes.INPUT
+        assert hw.get_pin_mode(13) == PinModes.INPUT
+        assert hw.get_pin_mode(16) == PinModes.INPUT
+        assert hw.get_pin_mode(4) == PinModes.INPUT
+        assert hw.get_pin_mode(14) == PinModes.OUTPUT
+        assert hw.get_pin_mode(35) == PinModes.INPUT
+        assert hw.get_pin_mode(17) == PinModes.INPUT
+        assert hw.get_pin_mode(34) == PinModes.OUTPUT
```

Comment:
In gpio.h, pin 3 - buzzer, is also listed but his doesn't seem to be set anywhere so I haven't included it